README.md

```diff
@@ -28,6 +28,19 @@ npm run serve
 
 You should now have the application running and accessible at <http://localhost:8080> (note: will run on a different port if 8080 is occupied, most common alternative port is 8081).
 
+## Building as web component
+To build the project as a web component, run the command `npm run build-component`. This will create a file in the `dist` folder named `license-chooser.min.js`. It can be used to load the web-component in any JS project. There is also a sample `demo.html` created.
+To be able to use the file it should either be rendered statically from the integrater's web-app or be published on a CDN. Following code can be used to integrate this in other apps:
+
+```
+<script src="https://unpkg.com/vue"></script>
+<script src="https://unpkg.com/license-chooser.js"></script>
```

Comment:
This will probably be namespaced as follows:

```suggestion
<script src="https://unpkg.com/@creativecommons/chooser/dist/license-chooser.js"></script>
```